package.json

```diff
@@ -54,11 +48,9 @@
     "marked": "^3.0.8",
     "prettier": "^2.8.8",
     "sass": "^1.79.4",
-    "sass-loader": "^10.5.2",
-    "typescript": "~4.1.5",
-    "vue-cli-plugin-i18n": "~2.1.2",
-    "vue-cli-plugin-vuetify": "^2.5.8",
-    "vue-template-compiler": "^2.7.16",
+    "typescript": "~5.6.3",
+    "unplugin-vue-components": "^0.27.4",
+    "vite": "^5.4.11",
```

Comment:
Not sure if worth swapping on this PR but Vite 6 has since been released
https://vite.dev/blog/announcing-vite6

Reply:
the vue 2 plugin does not support it yet

Reply:
👍